src/rust/engine/src/nodes/task.rs

```diff
@@ -48,8 +48,23 @@ impl Task {
         call: externs::Call,
     ) -> NodeResult<Value> {
         let context = context.clone();
+        let vtable_entries = context.core.tasks.vtable().get(&call.rule_id);
```

Comment:
Why not just `vtable_entries.and_then(|ve| ...)`  instead of matching?

Reply:
No particular reason, can switch to and_then().